### ISSUE TITLE
Not all .avi or .mp4 are shown

### ISSUE THREAD

**Issue by Cadencevr** [GoldenCheetah/GoldenCheetah#4235]
Hi, 
Nice piece of software.
I have a lot of .avi and .mp4 video files, but i noticed that a lot of my video files are not shown. Some are shown.
I don't know if this is a codec issue?

I seems to me that this is a bug, but maybe i'm wrong.

Regards
Klaasjan

Issue tracker is **only** for Bugs and Features, please don't open issues for questions or technical support. Before to open a new issue please read the contributing guidelines (link below).

If you have questions, please read the FAQs and User's/Developer's Guide:
* FAQs - https://github.com/GoldenCheetah/GoldenCheetah/wiki/FAQ
* User's Guide - https://github.com/GoldenCheetah/GoldenCheetah/wiki/UG_Main-Page_Users-Guide
* Developer's Guide - https://github.com/GoldenCheetah/GoldenCheetah/wiki/Developers-guide

If you need help or technical support please use the forums:
* Users - https://groups.google.com/forum/#!forum/golden-cheetah-users
* Developers - https://groups.google.com/forum/#!forum/golden-cheetah-developers
